### ISSUE TITLE
[Window] - Compilation: Error "getintegercoefficients.o"

### ISSUE THREAD

**Issue by Elkantor** [jart/cosmopolitan#716]
Following every steps in the readme for building on windows, I got this error : 

![image](https://user-images.githubusercontent.com/20202017/211161877-6707f2da-7c07-4a32-a812-bdbf304f7098.png)

**Comment by pkulchenko** [comment]
The error is in the text before the screenshot; can you include that as well?

**Comment by Elkantor** [comment]
Of course, here's the full steps in few screenshots for the build/boostrap/make.com command on a git bash on windows : 
![image](https://user-images.githubusercontent.com/20202017/211163206-9130fe44-ca29-4137-9b14-f015d802a379.png)
![image](https://user-images.githubusercontent.com/20202017/211163229-69586286-cf25-42e4-839a-774b12662d64.png)
![image](https://user-images.githubusercontent.com/20202017/211163255-15c4ef71-2edf-42de-92a8-6be05920851c.png)
![image](https://user-images.githubusercontent.com/20202017/211163274-35f5cc3c-a1e1-4dce-bd3a-39b9766a2cce.png)

**Comment by Elkantor** [comment]
I'm pretty sure the error comes from the lines juste after the error, as it says the error happen when compiling the getintegercoefficients.o object file ;)

**Comment by Elkantor** [comment]
Okay, after running the command line alone, here's what I got, seems like a compiling error on line 112 of this file: 
![image](https://user-images.githubusercontent.com/20202017/211163398-8effae82-16cf-433c-8fc6-0c91d3e9967b.png)

**Comment by Elkantor** [comment]
I had the flag 
```make
-Wno-attributes
```
as a quick "fix", even it's not really a fix, that works for compiling this file, now get an other error.
If anyone reading this issue has compiling successfully on windows, we could update the steps in the readme.

![image](https://user-images.githubusercontent.com/20202017/211164742-49f42ff8-dfda-44db-a00c-16a114901e08.png)

**Comment by Elkantor** [comment]
The new error:
![image](https://user-images.githubusercontent.com/20202017/211164875-88c3ae5f-6926-4abc-a07d-db2a5a4a392c.png)

With more details:
![image](https://user-images.githubusercontent.com/20202017/211164882-a320fe68-ddb9-4747-819d-fcf68180b047.png)

**Comment by Elkantor** [comment]
The problem seems to be that the include directive is not working in assembly files, because if I copy paste the assembly macros in each file, instead of including the macros.internal.h header file, it works.

![image](https://user-images.githubusercontent.com/20202017/211168601-03a0baa9-9e09-4cf4-b43c-95afb5044553.png)

**Comment by Elkantor** [comment]
Update:

Got it to work by replacing every:
```C
#include "libc/runtime/pc.internal.h"
```
In assembly files (512 in total), by: 
```C
.include "libc/runtime/pc.internal.h"
```

BUT, really weird, you need to not do that in the file libc\intrin\interrupts.S, for whatever reason.

So got to a point where there's an other issue, but I'm close to it I guess ;) 

![image](https://user-images.githubusercontent.com/20202017/211169511-fe09ca6e-d49c-4efd-ba03-0be03df3b3e3.png)

**Comment by tkchia** [comment]
Hello @Elkantor,

Hmm... the file is called `e2big.S`, not `E2BIG.s`, on my setup.  This is strange.  Perhaps something is weirding out because you are running the build on a case-insensitive filesystem.

`gcc` will call the C preprocessor when processing `.S` files, but will not call the C preprocessor when processing `.s` files.  The letter case matters.

Thank you!

**Comment by tkchia** [comment]
Hello @Elkantor,

On my setup there _is_ an `E2BIG.s`, but it is under `libc/sysv/consts/`, not `libc/sysv/errfuns/`.

(The former `E2BIG.s` defines the "constant" `E2BIG`.  The latter `e2big.S` is meant to define a function `e2big()` that sets `errno := E2BIG` and then returns `-1`.)

Thank you!
